lib/updaters/types/python.js

```diff
@@ -0,0 +1,29 @@
+const semverRegex = /version[" ]*=[ ]*["'](.*)["']/i
```

Comment:
This variable is a bit misleadingly named - it might be better as `versionExtractRegex` or similar, since it's not really testing for semver.